Documentation/spelling_wordlist.txt

```diff
@@ -720,6 +723,7 @@ roadmap
 rollout
 routable
 routingMode
+rst
```

Comment:
I don't understand why this is required? All I can find with `rst` is the include of `alpha.rst`, but we haven't required this spelling exception for `beta.rst` so far